go.mod

```diff
@@ -171,3 +171,7 @@ replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.15.5+incompatible
 
 replace github.com/banzaicloud/bank-vaults => github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
+
+replace github.com/spf13/cobra v0.0.0-00000000000000-c46add8a6528 => github.com/spf13/cobra v0.0.0-20170711120833-c46add8a6528
+
+go 1.13
```

Comment:
Not sure whether we can enforce 1.13 at this point. Effectively we are forcing everyone to go to 1.13 now, right?

@rawlingsj, I seem to recall you having a discussion around upgrading to Golang 1.13 and basically holding it back for now. Am I just imagining this?